coalib/bearlib/languages/definitions/VB.py

```diff
@@ -0,0 +1,12 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class VB:
+    aliases = 'vb',
+    extensions = '.vb',
+    comment_delimiter = "'"
+    string_delimiters = {'"': '"'}
+    multiline_string_delimiters = {'"': '"'}
+    encapsulators = {'(': ')', '{': '}'}
+    escape_quotes = '""',
```

Comment:
`escape_quotes` is not a property for language files yet, and very likely is not designed correctly.
I already rejected other patches previously because you included unaccepted properties to language files.

Reply:
Oops sorry, I copy pasted it, didn't realize this came with it. We probably can design it better. I'll remove this for now.